### ISSUE TITLE
Is Laravel Ui no longer supported?

### ISSUE THREAD

**Issue by ChristYoga123** [LaravelDaily/Larastarters#146]

**Comment by krekas** [comment]
What do you mean?

**Comment by krekas** [comment]
Can you find what was changed and make a pr then?

**Comment by gbretas** [comment]
i get installation stuckes at:

```
> @php artisan vendor:publish --tag=laravel-assets --ansi --force

   INFO  No publishable resources for tag [laravel-assets].  

No security vulnerability advisories found
```

**Comment by boedyirh** [comment]
Same here

**Comment by krekas** [comment]
Will anyone make a pr with the fix?

**Comment by gbretas** [comment]
I dont find the possible solution at this time.

**Comment by gbretas** [comment]
I have tried with old laravel version, and with 2.3.0 of larastarts and cant install.

**Comment by krekas** [comment]
Package always uses the latest laravel/ui. Please find the issue and make a pr.

**Comment by gbretas** [comment]
I already tried to change the version, but can`t fix via code, but found an solution to instalation...


```
composer create-project laravel/laravel MY_APP --prefer-dist
cd MY_APP 

composer require laraveldaily/larastarters --dev
composer require laravel/ui "4.2.2"

REMOVE LINE 131 OF vendor/laraveldaily/larastarters/src/Console/InstallCommand.php

php artisan larastarters:install```

**Comment by gbretas** [comment]
i`ll add some flag to instalation.

**Comment by erensertkaya** [comment]
i think some change in dependencies broke the package it's not working at all i even tried with laravel 9.

**Comment by gbretas** [comment]
See in my git the solution..

They don’t approve the pull

Em dom., 18 de fev. de 2024 às 14:02, Eren Sertkaya <
***@***.***> escreveu:

**Comment by krekas** [comment]
> See in my git the solution..
> 
> They don’t approve the pull
> 
 Because installation should be done with one command from this package. That was the idea from Povilas

**Comment by gbretas** [comment]
When I have some time today, I’ll try to fix that..

Em dom., 18 de fev. de 2024 às 14:13, Gustavo Bretas <
***@***.***> escreveu:

**Comment by PovilasKorop** [comment]
Fixed in v2.3.3
